src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/generate_proposals.hpp

```diff
@@ -26,6 +26,7 @@ class GenerateProposalsLayerTest :
         virtual public SubgraphBaseTest {
 protected:
     void SetUp() override;
+    void compare(const std::vector<ov::Tensor>& expected, const std::vector<ov::Tensor>& actual) override;
```

Comment:
the same as a previous comment. It helps to provide correct inputs for ANY graph

Reply:
Part of outputs can be zero. GenerateProposals needs custom compare function.

Reply:
@andrei-cv  Why could we extend the comparation map in the base class??